asm/f32/sdot.go

```diff
@@ -4,16 +4,16 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-package asm
+package f32
 
-func DsdotUnitary(x, y []float32) (sum float64) {
+func SdotUnitary(x, y []float32) (sum float64) {
```

Comment:
I don't think this is right. This is DSDOT, not SDOT. It should be in f63, but is should have a D prefix (or something).